<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GsExporterDockWidgetBase</class>
 <widget class="QDockWidget" name="GsExporterDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>556</height>
   </rect>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Gemeindescan Exporter</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="layerLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Map layer to export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsMapLayerComboBox" name="m_layer_cb"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-42</y>
         <width>658</width>
         <height>351</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QgsCollapsibleGroupBox" name="mGroupBox_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Snapshot settings</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="collapsed">
           <bool>false</bool>
          </property>
          <property name="scrollOnExpand">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Author</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="titleInput"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="authorInput"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Layer bounds</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>License</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="licenseInput"/>
           </item>
           <item row="5" column="1">
            <widget class="QTextEdit" name="descriptionInput"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="sourceInput"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Snapshot description</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="versionInput"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Version</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Source</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Snapshot title</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="1" colspan="2">
              <widget class="QPushButton" name="btn_calculate_extent">
               <property name="text">
                <string>Calculate bounds</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="sb_extent_precision"/>
             </item>
             <item row="0" column="0">
              <widget class="QLineEdit" name="le_extent"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Precision</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>About</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="metadataLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This tool exports Gemeindescan data package. Data
           package specification is based on the &lt;a href=&quot;https://frictionlessdata.io/&quot;&gt;&lt;span style=&quot;
           text-decoration: underline; color:#0000ff;&quot;&gt;Frictionless Data toolkit.&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Main
           development by &lt;a href=&quot;https://www.gispo.fi/en/home/&quot;&gt;&lt;span style=&quot; text-decoration:
           underline; color:#0000ff;&quot;&gt;Gispo Ltd &lt;/span&gt;&lt;/a&gt;and the tool is licensed under &lt;a
           href=&quot;https://www.gnu.org/licenses/gpl-3.0.en.html&quot;&gt;&lt;span style=&quot; text-decoration:
           underline; color:#0000ff;&quot;&gt;GPL v3 license.&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Gemeindescan
           exporter &lt;a href=&quot;https://github.com/GispoCoding/gemeindescan-exporter&quot;&gt;&lt;span style=&quot;
           text-decoration: underline; color:#0000ff;&quot;&gt;GitHub repository&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
          </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonReset">
        <property name="text">
         <string>Reset settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonExport">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
